src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -713,7 +713,7 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
                     argument = CreateParamArrayArgument(syntax, parameter.Type, paramArray.ToImmutableAndFree(), null, binder);
```

Comment:
Noticed that we are not setting `WasCompilerGenerated` property correctly for those synthetized bound nodes in this file.  But I guess this is a separate issue.